src/mango/src/mango_idx_view.erl

```diff
@@ -34,6 +34,7 @@
 -include_lib("couch/include/couch_db.hrl").
 -include("mango.hrl").
 -include("mango_idx.hrl").
+-include("mango_idx_view.hrl").
```

Comment:
I've only had a very quick look at this PR. But I'm not sure why the mango indexes are affected? From our discussion, we agreed that map only indexes wouldn't use ebtree?